tests/shared/src/test/scala/scala/meta/tests/parsers/CommonTrees.scala

```diff
@@ -27,7 +27,7 @@ trait CommonTrees {
   }
 
   object EmptyCtor {
-    def apply() = Ctor.Primary(Nil, Name.Anonymous(), Nil)
+    def apply() = Ctor.Primary(Nil, Name.Anonymous(), Seq.empty[Term.ParamClause])
```

Comment:
why not ctor (defined below)?

Reply:
Went the opposite way, ctor now uses EmptyCtor